de.metas.handlingunits.client/src/main/java/de/metas/handlingunits/client/terminal/select/api/impl/ReceiptScheduleFiltering.java

```diff
@@ -159,7 +159,6 @@ public boolean accept(final I_M_ReceiptSchedule schedule)
 				.stream()
 
 		// task 06570: exclude lines that are packaging materials
-		// TODO: make isPackagingMaterial a physical column https://github.com/metasfresh/metasfresh/issues/388
 				.filter(rs -> !rs.isPackagingMaterial())
```

Comment:
with IsPackagingMaterial beeing a physical column, we don't need this filter anymove.
Instead, please add an equalsfilter to the querybuilder